metricflow/test/integration/test_cases/itest_semi_additive_measure.yaml

```diff
@@ -29,6 +29,40 @@ integration_test:
     GROUP BY
       a.user
 ---
+integration_test:
+  name: semi_additive_measure_query_with_identifier_grouping_no_group_by
+  description: |
+    Tests selecting a measure with semi additive properties with a window_grouping that are not in the query group by
+  model: SIMPLE_MODEL
+  metrics: ["current_account_balance_by_user"]
```

Comment:
nit: We should probably rename this to something like `total_current_user_account_balances` or something.